ecs-cli/modules/cli/servicediscovery/servicediscovery_helper.go

```diff
@@ -242,9 +242,14 @@ func truncate(s string, length int) string {
 	return s
 }
 
-func getSDSCFNParams(namespaceID, sdsName, networkMode string, input *utils.ServiceDiscovery) *cloudformation.CfnStackParams {
+func getSDSCFNParams(namespaceID, ecsServiceName, networkMode string, input *utils.ServiceDiscovery) *cloudformation.CfnStackParams {
 	cfnParams := cloudformation.NewCfnStackParams(requiredParamsSDS)
 
+	sdsName := input.ServiceDiscoveryService.Name
+	if sdsName == "" {
```

Comment:
Can this ever be nil?

Reply:
No

Reply:
https://github.com/aws/amazon-ecs-cli/blob/servicediscovery/ecs-cli/modules/cli/servicediscovery/servicediscovery_app.go#L51